src/Quartz.Tests.Integration/Quartz.Tests.Integration.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;$(TargetFullFrameworkVersion)</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;$(TargetFullFrameworkVersion)</TargetFrameworks>
```

Comment:
While convenient and useful, testing against .NET Core 2.1 is not mandatory. Let me know if you want me to revert this change.

Reply:
I think this is fine for now.